src/platforms/ios/ioscontroller.swift

```diff
@@ -254,11 +256,45 @@ public class IOSControllerImpl : NSObject {
             }
         }
     }
+    @objc func setAlwaysOn(setEnabled: Bool){
```

Comment:
Currently, nothing in the code explicitly says this is for "connect on reboot". It seems like this function might be a good spot. Can you add a quick comment here (or another spot that seems appropriate)?